modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/BaseServerComponent.java

```diff
@@ -143,6 +144,12 @@ public AvailabilityType getAvailability() {
                 previousAvailabilityType = DOWN;
                 availabilityType = DOWN;
                 securityRealmReady = false;
+            } catch (UnauthorizedException e) {
+                previousAvailabilityType = DOWN;
+                availabilityType = DOWN;
+                throw new InvalidPluginConfigurationException("User authenticated via HTTP management interface does"
```

Comment:
Le 02/06/2015 10:47, Libor Zoubek a écrit :
> Catch block rethrowing IPCE is needed anyway, because
> |ASConnection.execute()| throws it when credentials are incorrect

Mmm, I don't think it's needed. If ASConnection#execute throws IPCE 
because of invalid credentials then we should just let the exception 
bubble up.